test/model-definition.test.js

```diff
@@ -375,13 +375,10 @@ describe('ModelDefinition class', function() {
     });
   });
 
-  it('should report deprecation warning for property names containing dot', function() {
-    var message = 'deprecation not reported';
-    process.once('deprecation', function(err) { message = err.message; });
-
-    memory.createModel('Dotted', { 'dot.name': String });
-
-    message.should.match(/Dotted.*dot\.name/);
+  it('should throw error for property names containing dot', function() {
+    (function() { memory.createModel('Dotted', { 'dot.name': String }); })
+      .should
+      .throw(/(dot\(s\)).*Dotted.*dot\.name/);
```

Comment:
I think those two brackets creating a capture group are not needed. Unless I am missing something?

```js
/dot\(s\).*Dotted.*dot\.name/
```